index.html

```diff
@@ -266,5 +266,32 @@ <h3 class="skillsTitle">Skills</h3>
   </div>
 </section>
 
+            <!-- This is the contact form -->
+
+<section class="contact" id="contactForm">
+    <h2 class="contactHeader">Contact me</h2>
+    <p class="contactDescription">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+    <form action="https://formspree.io/f/mgedkave" class="contactForm" method="post">
+      <ul class="contactField">
+        <li>
+          <label for="name"></label>
+          <input class = "contactName" type="text" id="name" name="name" placeholder="Enter your name here ..." maxlength="30" required />
+        </li>
+        <li>
+          <label for="email"></label>
+          <input class="emailAddress" type="email" id="email" name="email" placeholder="Enter your e-mail here ..." required />
+        </li>
+        <li>
+          <label for="message"></label>
+          <textarea class="contactMessage" id="message" placeholder="Type your message here ..." maxlength="500" required></textarea>
+        </li>
+        <li class="formSubmit">
+          <input type="submit" class="submitButton" value="Get In Touch" />
+        </li>
```

Comment:
- [ ] You can change the  on hover effect color  for the input button to look like the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A2238)
#

Reply:
The colors are the same on the hover effect.. maybe check the CSS file as well. For the hover effect to take place, you need to see the page in Desktop mode because you can't really hover your finger on your phone's screen, can you?

Reply:
They actually aren't.
This is the hover effect.
![hover](https://user-images.githubusercontent.com/90456900/154096903-db6c37ca-3ad4-4842-b2c6-3bbb6f20d8db.png)
 
This is the pressed effect.
![pressed](https://user-images.githubusercontent.com/90456900/154097011-04e61f5d-23aa-460a-8d8d-123575b274e6.png)

They are nowhere near the same color. I was thinking it might be an issue with my monitor, maybe some color issue because yea, they happen, but I used my 2nd monitor and they are still different. @prolajumokeoni